reclaim/src/main/java/com/fueled/reclaim/BaseItem.java

```diff
@@ -146,4 +147,35 @@ public T3 getViewHolder() {
 
         return null;
     }
+
+    /**
+     * Called by the DiffChecker to decide whether two object represent the same Item.
+     * <p>
+     * For example, if your items have unique ids, this method should check their id equality.
+     *
+     * @param newItem the other item to be checked.
+     * @return True if the two items represent the same object or false if they are different.
+     */
+    public boolean isTheSame(BaseItem newItem) {
+        return false;
```

Comment:
@julien-fueled this is because by default we assume the items are different, as implementing this check is the responsibility of the child class extending `BaseItem`.